xcodeproj/internal/runner.template.sh

```diff
@@ -101,11 +102,7 @@ fi
 
 installer_flags+=(--bazel_path "$bazel_path")
 
-if [[ %is_fixture% -eq 1 ]]; then
-  execution_root=$("$bazel_path" info --config=fixtures execution_root)
-else
-  execution_root=$("$bazel_path" info execution_root)
-fi
```

Comment:
This technically speeds up generation a bit (super minor), since the file is cached